app/views/TransportMapper.scala

```diff
@@ -0,0 +1,35 @@
+/*
+ * Copyright 2020 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package views
+
+import javax.inject.Singleton
+import models.notifications.queries.Transport
+import services.Countries.countryName
+import uk.gov.hmrc.govukfrontend.views.viewmodels.content.{Content, Empty, Text}
+
+@Singleton
+class TransportMapper {
+
+  def transportHtml(transport: Transport): Content =
+    (transport.transportId, transport.nationality) match {
+      case (Some(id), Some(nationality)) => Text(s"$id, ${countryName(nationality)}")
+      case (Some(id), None)              => Text(s"$id")
+      case (None, Some(nationality))     => Text(s"${countryName(nationality)}")
+      case _                             => Empty
+    }
```

Comment:
Could it be just translating country (if exists) and combining with id?
I mean different implementation using for e.g. for comprehension? My first impresion was "Why so many cases?" ;)

Reply:
Both the id and nationality are optional and if they are both there I want a comma between them.  This was the first idea I had.  There may be others.

Reply:
```
def transportHtml(transport: Transport): Content =
    Text((transport.transportId.toSeq ++ transport.nationality.map(countryName(_)).toSeq).mkString(", "))
```